### ISSUE TITLE
UI completely bricks when trying to create pool

### ISSUE THREAD

**Issue by markusdd** [OpenMediaVault-Plugin-Developers/openmediavault-zfs#62]
Hi,

I'm currently evaluating OMV 5 with ZFS plugin and created a VM with 6 disks to be formed into a 3 mirror pool. GUI has no option for that, so create one pool with 1 mirror and then try to add second mirror by trying the 'add' option.
Although I choose 2 drives (and as per zpool status the operation worked) the GUI form then on is completely broken, always answering with this:

![grafik](https://user-images.githubusercontent.com/25175069/71562494-31085980-2a81-11ea-9425-c2e3bd593a91.png)

The pool itself is completely fine:

```
root@openmediavault:~# zpool status
  pool: tank
 state: ONLINE
  scan: none requested
config:

        NAME                                       STATE     READ WRITE CKSUM
        tank                                       ONLINE       0     0     0
          mirror-0                                 ONLINE       0     0     0
            ata-VBOX_HARDDISK_VBd58ebfc4-3984e4c4  ONLINE       0     0     0
            ata-VBOX_HARDDISK_VB65a3b330-0239a696  ONLINE       0     0     0
          mirror-1                                 ONLINE       0     0     0
            ata-VBOX_HARDDISK_VB0a4dd0e2-7f72f9cd  ONLINE       0     0     0
            ata-VBOX_HARDDISK_VB27045e77-efde787e  ONLINE       0     0     0

errors: No known data errors
```

I then destroyed the pool again and created it completely on the command line

```
zpool create tank mirror ata-VBOX_HARDDISK_VBd58ebfc4-3984e4c4 ata-VBOX_HARDDISK_VB65a3b330-0239a696 mirror ata-VBOX_HARDDISK_VB0a4dd0e2-7f72f9cd ata-VBOX_HARDDISK_VB27045e77-efde787e mirror ata-VBOX_HARDDISK_VB14d81a8e-14a51a02 ata-VBOX_HARDDISK_VB84a35525-4a20e8b6
```

which worked fine:

```
root@openmediavault:~# zpool status
  pool: tank
 state: ONLINE
  scan: none requested
config:

        NAME                                       STATE     READ WRITE CKSUM
        tank                                       ONLINE       0     0     0
          mirror-0                                 ONLINE       0     0     0
            ata-VBOX_HARDDISK_VBd58ebfc4-3984e4c4  ONLINE       0     0     0
            ata-VBOX_HARDDISK_VB65a3b330-0239a696  ONLINE       0     0     0
          mirror-1                                 ONLINE       0     0     0
            ata-VBOX_HARDDISK_VB0a4dd0e2-7f72f9cd  ONLINE       0     0     0
            ata-VBOX_HARDDISK_VB27045e77-efde787e  ONLINE       0     0     0
          mirror-2                                 ONLINE       0     0     0
            ata-VBOX_HARDDISK_VB14d81a8e-14a51a02  ONLINE       0     0     0
            ata-VBOX_HARDDISK_VB84a35525-4a20e8b6  ONLINE       0     0     0

errors: No known data errors
```

Still, the GUI is completely broken with the above message and nothing else can be done in the ZFS tab.
Under the filesystem tab the zfs volume shows up fine:

![grafik](https://user-images.githubusercontent.com/25175069/71562605-c6f0b400-2a82-11ea-972e-6cebdd5161c4.png)

Additionally, when removing the pool on the command line (and the gui works again), all the drives which were used before don't show up as available in the pool creating, you all have to wipe them manually before they show up again.

All in all my first encounter was pretty bumpy you could say...

Versions:
```
root@openmediavault:~# zfs --version
zfs-0.8.2-3~bpo10+1
zfs-kmod-0.8.2-3~bpo10+1
```

OMV 5.2.1-1 (Usul)

**Comment by markusdd** [comment]
UPDATE:
Actually I think it has nothing to do with pool creation after fiddling a bit more.
I destroyed the pool and instead made the 6 disks into a raidz2.
And voila, I got my GUI back:
![grafik](https://user-images.githubusercontent.com/25175069/71562754-12a45d00-2a85-11ea-942e-6ef0d97228fc.png)

So bottom line is: the GUI cannot seem to handle a pool which contains multiple striped mirrors as toplevel vdevs. Seems like a serious bug to me.
